plugins/components/visible/Visible.cpp

```diff
@@ -18,9 +18,9 @@ Components::IComponent *buildDefault()
 }
 
 LIBRARY_ENTRYPOINT
-Components::IComponent *buildWithParams(bool isVisible)
+Components::IComponent *buildWithParams(uint8_t isVisible)
 {
-    return new Components::Visible(isVisible);
+    return new Components::Visible(!!isVisible);
```

Comment:
Does `!!` convert an uint_8 in bool ?

Reply:
Yes that's a good old trick from C: since `!` converts a 0 to a 1, and any non-zero to a 0, chaining two together (`!!`) converts any integer to 0 or 1, which converts safely to bool

Reply:
okay thanks for the explanation :saluting_face: